tests/posix/threads_base/src/pthread.c

```diff
@@ -57,13 +57,13 @@ static int barrier_return[N_THR_E];
 
 static void *thread_top_exec(void *p1)
 {
-	int i, j, id = (int) POINTER_TO_INT(p1);
+	int i, j, id = (int)POINTER_TO_INT(p1);
 	int policy;
 	struct sched_param schedparam;
 
 	pthread_getschedparam(pthread_self(), &policy, &schedparam);
```

Comment:
`pthread_getschedparam()` and `pthread_setschedparam()` functional tests belong in the `xsi_realtime_threads` testsuite.